### PR TITLE
Create resources

### PR DESCRIPTION
**Pull request by jcpunk**
If the tests work, (big if), this addresses #17

### CODE REVIEW

**Comment by jcpunk** [comment]
I fear I've no idea what is wrong with my syntax.....

File: spec/classes/udev_spec.rb [hunk 1]
```diff
@@ -79,6 +79,17 @@
       end
     end
 
+    describe 'rule parameter' do
+      let(:params) {{ rules: '99-foo.rules' => { 'content' => 'generic_rule' }}}
```
Comment (jhoblitt) [inline spec/classes/udev_spec.rb]: You have mixed ruby 1.8.7 / 1.9.3+ hash syntax in this line, which is confusing.  I suspect there is a missing a hash declaration.  Eg.

    {{ rules => { '99-foo.rules' => { 'content' => 'generic_rule' }}}}

**Comment by jcpunk** [comment]
Any idea how to fix the one test case that is failing?

**Comment by jhoblitt** [comment]
@jcpunk I've removed ruby 1.8.7 from the travis matrix.  Could you rebase?

**Comment by jcpunk** [comment]
rebased

**Comment by jhoblitt** [comment]
released as `2.3.0`: https://forge.puppet.com/jhoblitt/udev/2.3.0

@jcpunk I wanted to get this out because I sat on it for so long.  Could you take a stab at adding a blurb to the README on this feature?

**Pull request closed without merging**